packages/owl-bot/src/database.ts

```diff
@@ -15,12 +15,18 @@
 import admin from 'firebase-admin';
 import {OwlBotLock} from './config-files';
 import {Configs, ConfigsStore} from './configs-store';
+import {IMinimatch, Minimatch} from 'minimatch';
```

Comment:
I recollect adding a few extensions to Minimatch, because in some cases it didn't quite match my mental model of globs:

https://github.com/istanbuljs/test-exclude/blob/master/index.js#L134

Might be worth eyeballing some of the helpers in that library.

Reply:
Good to know.  We should document exactly which variant of glob we support.